src/models/thread.ts

```diff
@@ -73,7 +78,7 @@ export class Thread extends EventEmitter {
         if (this.ready) {
             this.client.decryptEventIfNeeded(event, {});
         }
-        this.emit("Thread.update", this);
+        this.emit(ThreadEvents.Ready, this);
```

Comment:
this seems to have gone from update to ready, is this an intentional change? definitely doesn't look like straight TSification

Reply:
That's a mistake, thanks for catching! Probably went a bit too far with replacements.